content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,187 @@
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+### DATABASE_URL
+
+`DATABASE_URL` is used to connect to your database. The url format can differ depending on the database it is connecting to.
+Prisma currently supports 4 types of database connectors, of which 3 use the `DATABASE_URL` environment variable. (SQLite sources a file path)
+
+- PostgreSQL
+- MySQL
+- SQLite
+- SQLServer
+
+Example using a PostgreSQL connector:
+
+```js file=.env
+DATABASE_URL = 'postgresql://USER:PASSWORD@HOST:PORT/DATABASE'
+```
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the `Prisma Client`.
+
+Example setting `Prisma Client` level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` strips colors from error messages. It's a truthy value.
+
+```js
+if (process.env.NO_COLOR) {
+  // Do Something
+}
+```
+
+### APPDATA
+
+`APPDATA` is used for accessing the windows root cache directory.
+
+```js
+if (process.env.APPDATA) {
+    return path.join(process.env.APPDATA, 'Prisma')
+}
+```
+
+### INIT_CWD
+
+`INIT_CWD` is the directory where `npm run` was executed. `Prisma` uses it to check for the `schema.prisma` file.
+
+```js
+if (process.env.INIT_CWD) {
+    // Do something
+}
+```
+
+### NO_PROXY
+
+`NO_PROXY` is a comma seperated list if hostnames, IP addresses or IP ranges that do not require a proxy.
+
+```terminal
+NO_PROXY=myhostname.com,10.11.12.0/16,172.30.0.0/16
+```
+
+### HTTP_PROXY
+
+`HTTP_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTP_PROXY=http://proxy.example.com
+```
+
+### HTTPS_PROXY
+
+`HTTPS_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTPS_PROXY=http://proxy.example.com
+```
+
+### PRISMA_HIDE_UPDATE_MESSAGE
+
+`PRISMA_HIDE_UPDATE_MESSAGE` is used to when checking if a `Prisma` update is cached. It's a truthy value.
+
+```js
+const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
+```
+
+### PRISMA_GENERATE_SKIP_AUTOINSTALL
+
+`PRISMA_GENERATE_SKIP_AUTOINSTALL` is used to when checking if the `prisma generate` command does an auto install.
+
+```js
+if (!process.env.PRISMA_GENERATE_SKIP_AUTOINSTALL) {
+    // Do Something
+}
+```
+
+### SKIP_GENERATE
+
+`SKIP_GENERATE` is used to determine if the `Prisma Client` generation should be skipped.
+
+```js
+if (!process.env.SKIP_GENERATE) {
+    // Do Something
+}
+```
+
+### BROWSER
+
+`BROWSER` is for the `Prisma Studio` to determine which broswer it should open in. It's also used as a flag when starting the studio from the CLI.
```

Comment:
```suggestion
`BROWSER` is for `Prisma Studio` to force which browser it should be open in, if not set it will open in the default browser. It's also used as a flag when starting the studio from the CLI.
```